client/packages/system/src/Patient/InsuranceView/columns.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ColumnAlign,
+  ColumnDescription,
+  ColumnFormat,
+  useColumns,
+} from '@openmsupply-client/common';
+
+import { PatientInsuranceFragment } from '../api/operations.generated';
+
+type Status = 'Active' | 'Inactive';
+
+export const useInsuranceColumns = () => {
+  const columns: ColumnDescription<PatientInsuranceFragment>[] = [
+    {
+      label: 'label.policy-number',
+      key: 'policyNumber',
+    },
+    {
+      label: 'label.provider-name',
+      key: 'providerName',
+      accessor: ({ rowData }) => rowData.policyNumberFamily,
```

Comment:
Once loader PR is in - this needs to be updated to be consuming the name from insurance provider instead